doc/whats_new/upcoming_changes/towncrier_template.rst.jinja2

```diff
@@ -1,13 +1,24 @@
 {% if render_title %}
 {% if versiondata.name %}
-{{ versiondata.name }} {{ versiondata.version }} ({{ versiondata.date }})
-{{ top_underline * ((versiondata.name + versiondata.version + versiondata.date)|length + 4)}}
+{{ versiondata.name }} {{ versiondata.version }})
+{{ top_underline * ((versiondata.name + versiondata.version)|length + 3)}}
 {% else %}
-{{ versiondata.version }} ({{ versiondata.date }})
-{{ top_underline * ((versiondata.version + versiondata.date)|length + 3)}}
+{{ versiondata.version }}
+{{ top_underline * ((versiondata.version)|length + 2)}}
 {% endif %}
 {% endif %}
 
+{% set months = {
+    '01': 'January', '02': 'February', '03': 'March', '04': 'April',
+    '05': 'May', '06': 'June', '07': 'July', '08': 'August',
+    '09': 'September', '10': 'October', '11': 'November', '12': 'December'
+} %}
+{% set date_parts = versiondata.date.split('-') %}
+{% set year = date_parts[0] %}
+{% set month = date_parts[1] %}
+{% set date_release_lexical = months[month] + ' ' + year %}
+**{{ date_release_lexical }}**
```

Comment:
Maybe something like this is a tiny bit simpler (not that much I agree)?
```suggestion
{% set month_names = {
    '01': 'January', '02': 'February', '03': 'March', '04': 'April',
    '05': 'May', '06': 'June', '07': 'July', '08': 'August',
    '09': 'September', '10': 'October', '11': 'November', '12': 'December'
} %}
{% set year, month, _ = versiondata.date.split('-') %}
{% set release_date = month_names[month] + ' ' + year %}
**{{ release_date }}**
```

For completeness I tried using `__import__` in the template and it turns out you were right, you can not do this :wink:

Reply:
but I was doing it with a normal `import`.